src/components/Avatar/Avatar.component.tsx

```diff
@@ -291,15 +291,14 @@ const Avatar: FC<AvatarProps> = ({
                 rings={16}
               />
             )}
-            {effects?.bloom && (
-              <Bloom
-                luminanceThreshold={effects?.bloom?.luminanceThreshold}
-                luminanceSmoothing={effects?.bloom?.luminanceSmoothing}
-                intensity={effects?.bloom?.intensity}
-                kernelSize={effects?.bloom?.kernelSize}
-                mipmapBlur={effects?.bloom?.mipmapBlur}
-              />
-            )}
+
+            <Bloom
+              luminanceThreshold={effects?.bloom?.luminanceThreshold}
+              luminanceSmoothing={effects?.bloom?.luminanceSmoothing}
+              intensity={effects?.bloom?.intensity}
+              kernelSize={effects?.bloom?.kernelSize}
+              mipmapBlur={effects?.bloom?.mipmapBlur}
+            />
```

Comment:
I think this doesn't fix the problem.

See line 275 which does check for effects objects.
I think that check is faulty since it doesn't address individual effects being enabled like AO and Bloom.

Ideally we would not render effect composer when no post processing is needed.